src/test/groovy/graphql/Issue938.groovy

```diff
@@ -0,0 +1,135 @@
+package graphql
+
+import graphql.language.IntValue
+import graphql.schema.Coercing
+import graphql.schema.DataFetcher
+import graphql.schema.DataFetchingEnvironment
+import graphql.schema.GraphQLScalarType
+import graphql.schema.idl.RuntimeWiring
+import graphql.schema.idl.TypeRuntimeWiring
+import graphql.validation.ValidationError
+import graphql.validation.ValidationErrorType
+import spock.lang.Specification
+
+class Issue938 extends Specification {
+
+    public static GraphQLScalarType GraphQLTimestamp = new GraphQLScalarType("Timestamp", "Timestamp", new Coercing() {
+        @Override
+        Object serialize(Object input) {
+            return input
+        }
+
+        @Override
+        Object parseValue(Object input) {
+            GregorianCalendar calendar = new GregorianCalendar()
```

Comment:
I hope this doesn't copy anyone ... maybe using the new java data (since java 8) might be better?